src/utils/get-ts-config.js

```diff
@@ -51,6 +52,26 @@ const getTsConfig = async (memo, fs, sourcePath, projectRoot) => {
     }
 };
 
+/**
+ * @param {string} jsoncString
+ * @param {string?} fileName
+ * @returns {any}
+ */
+function parseJsoncOrThrow(jsoncString, fileName){
+    /**
+     * @type {JSONC.ParseError[]}
+    */
+    let errors = [];
+    let parsingResult = JSONC.parse(jsoncString, errors, {allowTrailingComma: true});
+    if(errors.length > 0){
+        let {error: errorCode, offset} = errors[0];
+        let errName = JSONC.printParseErrorCode(errorCode);
+        console.error({errName, offset});
```

Comment:
Pretty sure console statements in providers will not reach the user - this line probably can be omitted